src/Aggregation/Percentiles.php

```diff
@@ -79,15 +81,4 @@ public function addPercent(float $percent): self
     {
         return $this->addParam('percents', $percent);
     }
-
-    /**
-     * Defines how documents that are missing a value should
-     * be treated.
-     *
-     * @return $this
-     */
-    public function setMissing(float $missing): self
```

Comment:
Same than https://github.com/ruflin/Elastica/pull/1876#discussion_r528979016
See https://3v4l.org/QspR2

As a BC break should be added to the corresponding section in the changelog.